src/pages/singleLineMap/index.tsx

```diff
@@ -162,36 +181,38 @@ const SingleLineMapPage = () => {
       </Grid>
 
       <div className="map-info">
-        <MapContainer center={position.loc} zoom={8} scrollWheelZoom={true}>
-          <TileLayer
-            attribution='&copy <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
-            url="https://tile-a.openstreetmap.fr/hot/{z}/{x}/{y}.png"
-          />
-
-          {filteredPositions.map((pos, i) => {
-            const icon = busIcon({
-              operator_id: pos.operator?.toString() || 'default',
-              name: agencyList.find((agency) => agency.operator_ref === pos.operator)?.agency_name,
-            })
-            return (
-              <Marker position={pos.loc} icon={icon} key={i}>
-                <Popup minWidth={300} maxWidth={700}>
-                  <BusToolTip position={pos} icon={busIconPath(operatorId!)} />
-                </Popup>
-              </Marker>
-            )
-          })}
-
-          {paths.map((path) => (
-            <Polyline
-              key={path.vehicleRef}
-              pathOptions={{
-                color: getColorByHashString(path.vehicleRef.toString()),
-              }}
-              positions={path.locations.map(({ lat, lon }) => [lat, lon])}
+        <Button type="primary" className="expand-button" shape="circle" onClick={() => setIsExpanded(!isExpanded)} icon={<ExpandAltOutlined />} />
+        <ExpandedContainer>
```

Comment:
```suggestion
        <ExpandedContainer className={isExpanded ? 'expanded' : 'collapsed'}>
```
will let you use CSS to control the container design